modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,196 @@
+| *Summary*: XDCR cannot handle data larger than 20MB
+
+|===
+
+=== Query Service
+
+[#table_fixedissues_v704-queryservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51261[MB-51261^]
+| *Summary*: Incorrect result when Element references an alias in the query
+
+| https://issues.couchbase.com/browse/MB-51431[MB-51431^]
+| *Summary*: Avoid circular link in annotated value hierarchy
+
+| https://issues.couchbase.com/browse/MB-51447[MB-51447^]
+| *Summary*: select executing a function executing a select returns the same results
+
+| https://issues.couchbase.com/browse/MB-51453[MB-51453^]
+| *Summary*: IN/NOT IN filters not using Hash for evaluation
+
+| https://issues.couchbase.com/browse/MB-51455[MB-51455^]
+| *Summary*: Aggregates wrongly pushed
+
+| https://issues.couchbase.com/browse/MB-51578[MB-51578^]
+| *Summary*: Error 4321 generated when OR clause has a ($parameter is null) arm
+
+| https://issues.couchbase.com/browse/MB-51581[MB-51581^]
+| *Summary*: JOIN query not covered
+
+| https://issues.couchbase.com/browse/MB-516291[MB-51629^]
+| *Summary*: Negative integer in the 64bit range causes rounding
+
+| https://issues.couchbase.com/browse/MB-51739[MB-51739^]
+| *Summary*: LEFT JOIN breaks with between operator on non-existing attribute
+
+| https://issues.couchbase.com/browse/MB-52281[MB-52281^]
+| *Summary*: adhoc query index selection issue with LIKE as index condition and query parameters
+
+|===
+
+=== Index Service
+
+[#table_fixedissues_v704-indexservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-50861[MB-50861^]
+| *Summary*: Query log has flood with messages
+
+| https://issues.couchbase.com/browse/MB-51023[MB-51023^]
+| *Summary*: ns_server stop responding after continuous CRUD on bucket and collections
+
+| https://issues.couchbase.com/browse/MB-52000[MB-52000^]
+| *Summary*: Alter Index failure when nodes have un encrypted ports in encryption mode all
+
+|===
+
+=== Search Service
+
+[#table_fixedissues_v704-searchservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-50946[MB-50946^]
+| *Summary*: bleveMaxClauseCount reset to 1024 after restart
+
+|===
+
+=== Analytics Service
+
+[#table_fixedissues_v704-analyticsservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51676[MB-51676^]
+| *Summary*: To investigate the crash in MB-49471
+
+| https://issues.couchbase.com/browse/MB-51483[MB-51483^]
+| *Summary*: Data is not completely ingested into datasets
+
+| https://issues.couchbase.com/browse/MB-51482[MB-51482^]
+| *Summary*: Node is not upgraded and stuck trying to register with CC
+
+| https://issues.couchbase.com/browse/MB-51488[MB-51488^]
+| *Summary*: Ignore byte order marks when parsing external data
+
+| https://issues.couchbase.com/browse/MB-51700[MB-51700^]
+| *Summary*: CBAS service keeps crashing when one of the CBAS nodes is failed over
```

Comment:
It's called the Analytics Service in all our documentation and UI etc.

Reply:
ok